src/main/kotlin/com/bridgecrew/CheckovResult.kt

```diff
@@ -58,8 +60,26 @@ fun groupResultsByResource(results: ArrayList<CheckovResult>, project: Project,
 }
 
 fun getFailedChecksFromObj(resultsObj: JSONObject): ArrayList<CheckovResult> {
+    try {
+        val failedRaw = resultsObj.get("failed")
+        if (failedRaw == 0) {
+            throw CheckovResultException("Results empty")
+        }
+        } catch (e : JSONException) { }
+    try {
+        val summary = resultsObj.getJSONObject("summary")
+            val failedSummary = summary.get("failed")
+            val parsingErrorSummary: Int = summary.getInt("parsing_errors")
+            if (parsingErrorSummary > 0 ) {
+                throw CheckovResultParsingException("Checkov parsing error")
+            }
+            if (failedSummary == 0){
+                throw CheckovResultException("Results empty")
+            }
+    } catch (e : JSONException) { }
```

Comment:
not sure I understand this. why throw exception and then catch and do nothing ?

Reply:
Removed it , it was unnecessery